### ISSUE TITLE
a-Shell and a-Shell mini have the same folder name in File.app!

### ISSUE THREAD

**Issue by cgand** [holzschu/a-shell#451]
I’ve installed both a-Shell and a-Shell mini and in File.app there are two folders with the same name.
In screenshot is what I see when I try to pickFolder and I’ve to select the right one.
Should a-Shell mini folder be named “a-Shell mini”?
![64B43474-3D0F-4FCD-A6D2-8D8272E74632](https://user-images.githubusercontent.com/5794213/186770954-faac6fbb-ced2-47da-a2e2-a481a86c5a46.jpeg)

**Comment by holzschu** [comment]
I agree with you, I'm not sure how to do it properly. On iCloud, they share the same folder, which complicates things (so you can have a-Shell on your iPad and a-Shell mini on your iPhone, for example, and they share files). I figured few people would install both apps.

**Comment by cgand** [comment]
I see, maybe they should share the same folder on both iPad and iCloud. This could make things easier. But I don’t know how to manage the transition.

**Comment by holzschu** [comment]
That, on the other hand, is impossible: two different apps have different local folders (but they can share the same iCloud folder).

**Comment by cgand** [comment]
If apps must have two different local folders perhaps they should also have two different names.
“A-Shell” and “a-Shell mini”on local folder and only “a-Shell” on iCloud. But I know it could create confusion from a other point of view.
But, as you say, perhaps I’m one of the few with both app installed :-)

**Comment by personalizedrefrigerator** [comment]
I also have both apps installed (I find it useful for multitasking, as VimShell has issues running some interactive commands like `python`).
